src/frame.js

```diff
@@ -0,0 +1,29 @@
+function Frame (seqNum) {
+        this.rolls = [];
+        this.frames = [];
+        this.seqNum = seqNum || 0;
+
+    Frame.prototype.totalRolls = function() {
+        // if (this.rolls.length === 3) {
+        //     this.rolls.splice(0, this.rolls.length)
+        // }
+        return (this.rolls.length);
+    };
+    Frame.prototype.storeRoll = function(roll) {
+        return (this.rolls.push(roll));
+    };
+
+    Frame.prototype.newFrame = function(updateFrame) {
+        if (this.rolls.length === 2) {
+            this.rolls.splice(0, this.rolls.length)
+            this.frames.push(updateFrame)
+        }
+        return (this.rolls.length);
```

Comment:
This function is returning the length of your rolls array, do you mean the length of your frames here?

Reply:
Ahhh, yes - basically I wanted a newFrame to reset the rolls to 0 every time we jumped into a new frame. Thanks for pointing that out, it would make more sense for it to return the frame rather than the rolls array...